app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(name=person["name"],
+                          age=person["age"])
+                   for person in people]
+    for person in people:
+        name = person["name"]
+        if "wife" in person and person["wife"]:
+            Person.people[name].wife = Person.people.get(person["wife"])
```

Comment:
You could perform this check at once using get
```suggestion
        if person.get("wife"):
            Person.people[name].wife = Person.people.get(person["wife"])
```
and improve the logic further avoiding double retrieval of key by value and using walrus operator
```suggestion
        if wife_name := person.get("wife"):
            Person.people[name].wife = Person.people.get(wife_name)
```